pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -918,6 +926,35 @@ func (ctrl *PersistentVolumeController) updateVolumePhaseWithEvent(volume *v1.Pe
 	return newVol, nil
 }
 
+// assignDefaultStorageClass looks up all storage classes and attempts to assign a storage class to the given claim.
+// The storage class assigned is always the default one and if there is none or there are more defaults the claim
+// will not get updated.
+func (ctrl *PersistentVolumeController) assignDefaultStorageClass(claim *v1.PersistentVolumeClaim) (*v1.PersistentVolumeClaim, error) {
+	// [Unit test set 15]
+	if claim.Spec.StorageClassName != nil {
+		klog.V(4).Infof("can not assign storage class to PersistentVolumeClaim[%s]: pvc already has StorageClass[%s]", claimToClaimKey(claim), *claim.Spec.StorageClassName)
```

Comment:
This makes it feel like an error, but it's not, is it?  Any PVC that correctly got a default will trigger this?  Or maybe reword so it doesn't seem like a failure?

Reply:
I removed the message as it's not too useful anyway.